M2/Macaulay2/packages/Msolve/EXA/Msolve-testing.m2

```diff
@@ -0,0 +1,65 @@
+needsPackage("Msolve", Configuration => {
```

Comment:
Why are these two files in the `EXA` subdirectory included in the PR?  They're not loaded by the main file.  Should they be removed, and any interesting code be moved to tests and/or documentation?

Reply:
One of them now houses TESTs.
The other is housing a test that takes longer and may have problems (at least in previous versions).